general/coding_standards.rst

```diff
@@ -42,6 +42,12 @@ Variables are initialized according to these general guidelines:
       which can be a scalar or a composite type are initialized with
       ``memset()`` in order to minimize the amount of future headache.
 
+      Source tree of optee_client_ Git component as a specific rule which
```

Comment:
I believe it's more complicated than this. Some structs can be safely (as in safe to not cause warnings) initialized with `{ 0 }`, but not as a universal initializer as it clearly isn't universal. Struct in which the first element is a scalar can be safely initialized `{ 0 }`, but for others it depends on the compiler as far a I can tell.

Reply:
In addition there is already a paragraph about optee_client above. I think it should be modified to recommend `{ 0 }` for structs where the first element is known to be a scalar, and `memset()` otherwise, no?

Reply:
thanks for the feedback.
I've updated to P-R somewhat using @jforissier phrasing.